appveyor.yml

```diff
@@ -0,0 +1,23 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: c:\projects\symfony
+
+environment:
+  global:
+    PHP: c:/PHP
+
+init:
+  - SET PATH=c:\php\;%PATH%
+
+install:
+  - cinst gow
+  - cinst php
+  - cinst composer
```

Comment:
yep, but gow for example also has an interactive installer, but the package installs just fine. I'll resort to the previous way.